### DISCUSSION TITLE
How can i test the changes that i make into my forked repo for example how can I test api or any thing using the changes I made

### DISCUSSION THREAD

**Discussion by ShreySinha02** [fastapi/fastapi#11127] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
..
```

### Description

trying to check if it work or not

### Operating System

Linux

### FastAPI Version

 0.109.2

### Pydantic Version

1.10.14

### Python Version

3.10.12

**Comment by pbasista** [comment]
If you would like to get some reasonable answers, please be more specific on what you need. A generic question like "how to test" will most likely only yield generic answers.

If you would like to test the fork of this repository, you can start by looking at the included pytest-based [tests](https://github.com/tiangolo/fastapi/tree/master/tests).

**Reply by ShreySinha02** [reply]
no i don't want to run test i want to create api using changes i made in my local machine

**Reply by pbasista** [reply]
> i want to create api ...

Then do so.

If something is stopping you, explain specifically what it is. Then you might be able to get some reasonable answers.
